### PR TITLE
pcs: use ruby 2.2

### PR DESCRIPTION
**Pull request by grueni**
add dependencies for bundler 
use environment variables for RUBY

### CODE REVIEW

File: components/cluster/pcs/Makefile [hunk 1]
```diff
@@ -14,6 +14,12 @@
 
 include ../../../make-rules/shared-macros.mk
 
+RUBY_VERSION = 2.2
+RUBY_LIB_VERSION = 2.2.0
+RUBY.2.2 = /usr/ruby/2.2/bin/ruby
+RUBY = $(RUBY.$(RUBY_VERSION))
+RUBY_VERSIONS = $(RUBY_LIB_VERSION)
```
Comment (pyhalov) [inline components/cluster/pcs/Makefile]: @xenol I think we should add part of these definitions to shared-macros.mk, and here use just RUBY_VERSION=2.2

File: components/cluster/pcs/Makefile [hunk 2]
```diff
@@ -14,6 +14,12 @@
...
+
 COMPONENT_NAME= pcs
 COMPONENT_VERSION= 0.9.154
```
Comment (pyhalov) [inline components/cluster/pcs/Makefile]: @grueni Please, bump COMPONENT_REVISION

File: components/cluster/pcs/Makefile (hunk 1 above)
Comment (xen0l) [reply components/cluster/pcs/Makefile]: yes, we should move it to sharec-macros.mk

File: components/cluster/pcs/Makefile (hunk 2 above)
Comment (xen0l) [reply components/cluster/pcs/Makefile]: @grueni COMPONENT_REVISION has to be bumped, so build server picks up the build

**Comment by pyhalov** [comment]
Should it be RUBY 2.2 or does it work with 2.3 ?

**Comment by pyhalov** [comment]
Switch to ruby 2.3 was merged as https://github.com/OpenIndiana/oi-userland/commit/e4475440970198870c1761897e9b2661f3a965d7 . Additional changes (if necessary) should be submitted as separate PR.

**Pull request closed without merging**